sycl/include/CL/sycl/detail/pi.h

```diff
@@ -58,10 +57,9 @@
 // TODO: we need a mapping of PI to OpenCL somewhere, and this can be done
 // elsewhere, e.g. in the pi_opencl, but constants/enums mapping is now
 // done here, for efficiency and simplicity.
```

Comment:
I guess we can drop this comment as well. Right?

Reply:
Yeap, that's right.